### PR TITLE
feat: support include statement

### PR DESCRIPTION
**Pull request by TennyZhuang**
Add support to include another slt file, which is very useful in data importing.

The syntax is not a part of standard sqllogictest, and we are not ready to stabilize it, so I mark it as hidden.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -215,6 +224,32 @@ pub fn parse(script: &str) -> Result<Vec<Record>, Error> {
...
 }
 
+/// Parse a sqllogictest file and link all included scripts together.
+#[doc(hidden)]
+pub fn parse_file(filename: &str) -> Result<Vec<Record>, Error> {
+    parse_file_inner(Path::new(filename))
+}
+
+fn parse_file_inner(path: &Path) -> Result<Vec<Record>, Error> {
+    dbg!(path);
```
Comment (fuyufjh) [inline src/lib.rs:234]:
```suggestion
```

**Review by fuyufjh — APPROVED** [review]
LGTM

File: src/lib.rs [hunk 2]
```diff
@@ -215,6 +224,31 @@ pub fn parse(script: &str) -> Result<Vec<Record>, Error> {
     Ok(records)
 }
 
+/// Parse a sqllogictest file and link all included scripts together.
+#[doc(hidden)]
+pub fn parse_file(filename: &str) -> Result<Vec<Record>, Error> {
+    parse_file_inner(Path::new(filename))
```
Comment (liurenjie1024) [inline src/lib.rs:230]: The path should be relative to source file which contains `include` directive.

**Review by wangrunji0408 — APPROVED** [review]
LGTM

File: src/lib.rs (hunk 2 above)
Comment (TennyZhuang) [reply src/lib.rs:230]: But we need an entry..

**Pull request merged**